OpenUtau/Views/MessageBox.axaml

```diff
@@ -39,8 +43,15 @@
             </ControlTemplate>
           </Setter>
         </Style>
+        <Style Selector="Expander">
+          <Setter Property="Margin" Value="0, 20, 0, 0" />
+          <Setter Property="Width" Value="560" />
+          <Setter Property="IsExpanded" Value="False" />
```

Comment:
I'd prefer it expanded by default. The stack trace was added so that users' screenshots are actually useful. If they screenshot without expanding it then you will have to ask them to do so, etc, etc, which wastes energy.